code/src/index.css

```diff
@@ -1,13 +1,118 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Helvetica";
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #121212;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+main {
+  width: 80%;
+  margin: 0 auto;
 }
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+h1 {
+  color: #fff;
+  font-size: 20px;
+  padding-bottom: 0.5rem;
+  border-bottom: 0.8px #6e6e6e solid;
+}
+
+.artistsName {
+  color: #a0a0a0;
+  font-size: 14px;
+  margin: 0;
+}
+
+.albumsWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 1.5rem;
+}
+
+.albumWrapper {
+  padding: 1.5rem;
+  background-color: #171717;
+  border-radius: 8px;
+}
+  
+.albumName {
+  color: white;
+  font-size: 14px;
+}
+
+.albumImageWrapper {
+  position: relative;
+  width: 100%;
+}
+
+.albumImage {
+  width: 100%;
+  border-radius: 8px;
+}
+
+.albumOverlay {
+  background-color: rgba(0, 0, 0, 0.6);
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: none;
+}
+
+.icon {
+  width: 8%;
+  height: 8%;
+}
+
+.playWrapper {
+  width: 18%;
+  height: 18%;
+}
+
+.playIcon {
+  width: 100%;
+  height: 100%;
+  opacity: 0.7;
+ }
+ 
+.playIcon:hover {
+  opacity: 1;
+}
+
+.playWrapper:hover {
+  width: 22%;
+  height: 22%;
+  transition: all 0.3s ease;
+}
+
+.albumImageWrapper:hover .albumOverlay {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.albumName:hover, .artistName:hover {
+  text-decoration: underline;
+}
+
+/* ------------------ MEDIA QUERIES ------------------ */
+
+@media screen and (min-width: 768px) {
+  .albumsWrapper {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .albumsWrapper {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
```

Comment:
Your css code is very nicely structured and I have no comments other than that it looks great!